src/Page/Auth.elm

```diff
@@ -603,6 +610,19 @@ viewAccount { user } profileForm formErrors =
                 , label [ class "form-check-label", for "emailOptin" ]
                     [ text "J’accepte de recevoir des informations de la part d'Ecobalyse par email."
```

Comment:
Nit: not introduced by this patch, but the second apostrophe is not a French typography one

<img width="1000" height="500" alt="Image" src="https://github.com/user-attachments/assets/2ea60df5-dfd7-4b63-8064-4f2815839d39" />